src/renderer/item.store.ts

```diff
@@ -27,6 +27,10 @@ export abstract class ItemStore<T extends ItemObject = ItemObject> {
     return this.items.find(item => item.getName() === name);
   }
 
+  getIndex(item: T): number {
+    return this.items.findIndex(i => i === item);
+  }
```

Comment:
A more general form would be more useful
```suggestion
  getIndexById(id: string): number {
    return this.items.findIndex(i => i.getId() === id);
  }
```